### PR TITLE
Adding Fireperf to Firehorn

### PR DESCRIPTION
**Pull request by jposuna**

### CODE REVIEW

File: smoke-tests/src/main/java/com/google/firebase/testing/PerformanceMonitoringTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    networkRequest.setResponsePayloadSize(1024);
+    networkRequest.setHttpResponseCode(200);
+    networkRequest.setResponseContentType("text/html");
+    networkRequest.putAttribute("is_test", "true");
+
+    networkRequest.stop();
+
+    assertThat(networkRequest.getAttribute("is_test")).isEqualTo("true");
+  }
+}
```
Comment (yifanyang) [inline smoke-tests/src/main/java/com/google/firebase/testing/PerformanceMonitoringTest.java:61]: Missing a new line here, but it's up to you.

**Review by yifanyang — APPROVED** [review]

File: smoke-tests/src/main/java/com/google/firebase/testing/PerformanceMonitoringTest.java (hunk 1 above)
Comment (jposuna) [reply smoke-tests/src/main/java/com/google/firebase/testing/PerformanceMonitoringTest.java]: Done.

**Review by zijianjoy — APPROVED** [review]

**Review by ramanpreetSinghKhinda — APPROVED** [review]

**Pull request merged**